clang/test/CodeGenSYCL/int_header_esimd.cpp

```diff
@@ -0,0 +1,60 @@
+// RUN: %clang_cc1 -fsycl -fsycl-explicit-simd -fsycl-is-device -fsycl-int-header=%t.h %s
+// RUN: FileCheck -input-file=%t.h %s
+
+// This test checks that
+// 1) New isESIMD() member is generated into the integration header
+// 2) It returns 1 for ESIMD kernels and 0 - for non-ESIMD.
+
+// -- ESIMD Lambda kernel.
+template <typename ID, typename F>
+void kernelA(F f) __attribute__((sycl_kernel)) {
+  f();
+}
```

Comment:
Same here, please use queue::submit and single_task from the mock header included like system one.

Reply:
ok